assets/js/components/Sessions/CostHistoryChart.vue

```diff
@@ -285,10 +288,11 @@ export default {
 						grid: { display: false },
 						ticks: {
 							color: colors.muted,
-							callback: (value) =>
-								this.period === PERIODS.YEAR
-									? this.fmtMonth(new Date(this.year, value, 1), true)
-									: value,
+							callback: function (value) {
+								return vThis.period === PERIODS.YEAR
```

Comment:
Ich nehme für shadowing immer `that` 😀

Reply:
Ja, ich hab mich hier fürs `vue` Präfix entschieden. Machts für mich etwas klarer als ein generisches `that`. Glücklicherweise haben wir das bislang noch nirgendwo anders in der Codebasis gebraucht.